R/difftimes.r

```diff
@@ -79,13 +79,13 @@ make_difftime <- function(num = NULL, units = "auto", ...) {
   units <- standardise_date_names(units[[1]])
   switch(units,
          second = structure(num, units = "secs", class = "difftime"),
```

Comment:
Was this indention a manual tweak?

Reply:
No, since I used `scope = "spaces`, indention was not touched.